winml/test/common/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <unordered_map>
+#include <gtest/gtest.h>
+
+#include "runtimeParameters.h"
+
+namespace RuntimeParameters
+{
+    std::unordered_map<std::string, std::string> Parameters;
+}
+
+int main(int argc, char **argv)
```

Comment:
>main [](start = 4, length = 4)

can you add some comments in this file? would be good to explain how this stuff works